### DISCUSSION TITLE
Allow copy dashboard folder

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#38513] [Dashboards, Panels & Visualizations]
Dashboard folder can be renamed and deleted.

How about making the folder duplicatable(can be copied). So every dashboard inside one folder can be copied to a new folder in batch.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
